File: example_problems/hello/submissions/accepted/extra-file-js/test-hello.js
// This should give CORRECT on the default problem 'hello'.
// It used to give COMPILER-ERROR since it includes a random extra file.
//
// @EXPECTED_RESULTS@: CORRECT

if ( process.env.DOMJUDGE ) {
  console.log('Hello world!');
} else {
  console.log('DOMJUDGE not defined');
  process.exit(1);
}

File: example_problems/hello/submissions/accepted/extra-file-pas/test-hello.pas
(*
 * This should give CORRECT on the default problem 'hello',
 * It used to give COMPILER-ERROR since it includes a random extra file.
 *
 * @EXPECTED_RESULTS@: CORRECT
 *)

program helloworld(input, output);

var
   hello : string;

begin
   hello := 'Hello world!';
   writeln(hello);
end.
